### PR TITLE
Italian cheatsheet fix

### PR DESCRIPTION
**Pull request by hchienjo**
###### Improvement: **`Italian Cheat sheet: Minor typo fix`**

---
Fix wrong spelling of vegetarian.

Instant Answer Page: https://duck.co/ia/view/italian_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @marcostudios

### CODE REVIEW

**Comment by daxtheduck** [comment]
@hchienjo Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/perl.json [hunk 1]
```diff
@@ -132,7 +132,7 @@
         }],
         "Basic Syntax": [{
             "val": "Read command line params",
-            "key": "($a, $b) = shift(@ARGV);"
+            "key": "($a, $b) = (@ARGV);"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/perl.json]: Looks like this got included by mistake?

Comment (hchienjo) [inline share/goodie/cheat_sheets/json/perl.json]: @GuiltyDolphin, you are right. I must've missed it.

Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/perl.json]: @hchienjo We should remove it :)

Comment (hchienjo) [inline share/goodie/cheat_sheets/json/perl.json]: To be honest, this is my second pull request and I have no idea how to remove the second file.
Is it possible for you to guide me through it?

Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/perl.json]: @hchienjo If you do `git checkout master PATH_TO_FILE` you're working copy will then match `master`.

You can then commit that so your local copy matches master.

Comment (hchienjo) [inline share/goodie/cheat_sheets/json/perl.json]: Will this not interfere the other pull requet involving the second file? I mean this one 
https://github.com/duckduckgo/zeroclickinfo-goodies/pull/2972

Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/perl.json]: @hchienjo Nawp, they are on separate branches.

Comment (hchienjo) [inline share/goodie/cheat_sheets/json/perl.json]: @GuiltyDolphin Done. Thanks for the guidance.

**Comment by marcostudios** [comment]
Thanks for the fix :+1:

**Comment by GuiltyDolphin** [comment]
@hchienjo Thanks! :dolphin:

**Pull request merged**